settings.py

```diff
@@ -2256,6 +2256,16 @@ def _determine_couch_databases(couch_databases):
     'test-pna': 'custom.yeksi_naa_reports',
 }
 
+OTHER_DOMAINS_WITH_CUSTOM_MODULES = [
+    'fri',
+    'hope',
+    'logistics',
```

Comment:
we can remove this entry too, the custom stuff in this module used to cover the ews and ils projects, which are already in the migration blacklist

Reply:
no, that seems unrelated